### ISSUE TITLE
Unsafe

### ISSUE THREAD

**Issue by tbrown19** [mjbuchman/ff-league-history#33]
I'm sure you know this already but this is technically super unsafe to do
https://github.com/mjbuchman/ff-league-history/blob/0790d1c1a84bd0b71d9f8d22d9d63af43ec9b27f/server/server.js#L31

**Comment by mjbuchman** [comment]
Yeah I threw a lot of stuff together in the beginning so I could just get started and never went back to it. Exactly why my next big thing is clean up
